stix2elevator/convert_pattern.py

```diff
@@ -506,7 +507,8 @@ def make_constant(obj):
         return stix2.StringConstant(obj.strip())
     elif isinstance(obj, list):
         return stix2.ListConstant([make_constant(x) for x in obj])
-    # TODO: Timestamp
+    elif isinstance(obj, datetime.datetime) or isinstance(obj, stixmarx.api.types.MarkableDateTime):
```

Comment:
This could have been written as `issubclass(obj, datetime.datetime)` and I believe the same is true for other types if MarkableObjects from `stixmarx` become a problem.

Reply:
If nothing else, `isinstance` can take a tuple as its second argument, so you could write this as `if isinstance(obj, (datetime.datetime, stixmarx.api.types.MarkableDateTime))`